tests/Database/SampleFiles/SampleClass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace October\Rain\Tests\Database\SampleFiles;
```

Comment:
@iulianwebdev As this is a test fixure, could you instead move it to `tests/fixtures/Database/SampleClass.php` and update the namespace accordingly?

Reply:
Yes, I think it would make more sense. I'll try and update today.